actioncable/lib/action_cable/channel/test_case.rb

```diff
@@ -0,0 +1,275 @@
+# frozen_string_literal: true
+
+require "active_support"
+require "active_support/test_case"
+require "active_support/core_ext/hash/indifferent_access"
+require "json"
+
+module ActionCable
+  module Channel
+    class NonInferrableChannelError < ::StandardError
+      def initialize(name)
+        super "Unable to determine the channel to test from #{name}. " +
+          "You'll need to specify it using tests YourChannel in your " +
```

Comment:
Backticks around "tests YourChannel" to be clear that it's code to be added to the test case.

Reply:
👍 

Done